### PR TITLE
Audiowide font support and instructions on how to install on windows 10 stable

### PR DESCRIPTION
**Pull request by Chromico**
Audiowide support was added and also tested onto a real circuit board. The docs were updated to show how to install it in KiCad stable windows 10.

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,4 +1,4 @@
 [submodule "deps/Buzzard"]
 	path = deps/buzzard
-	url = https://github.com/VedantParanjape/Buzzard.git
+	url = https://github.com/Chromico/Buzzard.git
```
Comment (VedantParanjape) [inline .gitmodules]: This needs to be changed, right ?

File: README.md [hunk 2]
```diff
@@ -30,7 +32,7 @@ Ensure you also get the submodules with KiBuzzard
 
 ```console
 $ cd ~/.kicad/scripting/plugins
-$ git clone https://github.com/VedantParanjape/KiBuzzard.git --recursive
+$ git clone https://github.com/Chromico/KiBuzzard.git --recursive
```
Comment (VedantParanjape) [inline README.md]: Also this

File: requirements.txt [hunk 3]
```diff
@@ -1,6 +1,6 @@
-bezier==2020.5.19
```
Comment (VedantParanjape) [inline requirements.txt]: Let the version numbers be there.

**Comment by VedantParanjape** [comment]
Also changes to Buzzard don't belong here :) they belong either to my fork or the upstream repository. Can you remove that and only add the readme ?

File: .gitmodules (hunk 1 above)
Comment (Chromico) [reply .gitmodules]: Hold on lemme change that.

**Review by Chromico — COMMENTED** [review]
Made the changes and opened a PR on your Buzzard fork. Let me know if everything looks good :)

**Pull request closed without merging**